roles/bootstrap-os/files/bootstrap.sh

```diff
@@ -18,7 +18,11 @@ mv -n pypy-$PYPY_VERSION-linux64 pypy
 
 ## library fixup
 mkdir -p pypy/lib
-ln -snf /lib64/libncurses.so.5.9 $BINDIR/pypy/lib/libtinfo.so.5
+if [ -f /lib64/libncurses.so.5.9 ]; then
+  ln -snf /lib64/libncurses.so.5.9 $BINDIR/pypy/lib/libtinfo.so.5
+elif [ -f /lib64/libncurses.so.6.1 ]; then
```

Comment:
Don't you rather want to check and link the new version first and not check for the old version first.

In a case where libncurses 5.9 and 6.1 exists both?

Reply:
I checked on my coreos nodes. There is either one or the other. 
Very on fresh installations you will already find only the 6.1 version.

I was  running into issues because I was in an environment where some nodes updated and some nodes not, and I wanted to share my fix because I guess that people will run into the same issue soon.

Reply:
Yes. But I do tend to think that in a "programming way" it's best to check for the newest version first and then older. Yet it won't "do much" in this case.

Reply:
to me this is a matter of taste, if you consider the old behaviour as default and therefor check first or expect the new behaviour as default and check this first.
Anyways, if this is blocking it, I will change the order.